sycl/include/CL/sycl/INTEL/online_compiler.hpp

```diff
@@ -142,51 +175,48 @@ template <source_language Lang> class online_compiler {
 
   /// Target device stepping (implementation defined)
   std::string DeviceStepping;
-  // }
+
+  /// Handles to helper functions used by the implementation.
+  void *CompileToSPIRVHandle = nullptr;
+  void *FreeSPIRVOutputsHandle = nullptr;
 };
 
 // Specializations of the online_compiler class and 'compile' function for
 // particular languages and parameter types.
 
-/// Compiles given OpenCL source. May throw \c online_compile_error.
-/// @param src - contents of the source
+/// Compiles the given OpenCL source. May throw \c online_compile_error.
+/// @param src - contents of the source.
+/// @param options - compilation options (implementation defined); standard
+///   OpenCL JIT compiler options must be supported.
+template <>
+template <>
```

Comment:
Not clear why this double `template` and what is specialized here.

Reply:
The first template because the class is template class, the second template because the method is template. Removing any of those 2 lines will cause syntax error.